op-chain-ops/deployer/broadcaster/broadcaster.go

```diff
@@ -11,6 +11,7 @@ import (
 type Broadcaster interface {
 	Broadcast(ctx context.Context) ([]BroadcastResult, error)
 	Hook(bcast script.Broadcast)
+	Sender() common.Address
```

Comment:
I don't like this change as much. Broadcast structs can have different senders, the `vm.broadcast` family of cheatcodes allows you to mock different sender addresses throughout a script. There shouldn't be a single enshrined sender.

Reply:
I can rename to `Broadcaster()` if that helps. This is used to represent who the underlying private key is.

Reply:
But `script.Broadcast` already has a `From` field. Wouldn't that conflict, if the address does not match? Maybe a "default broadcaster" makes sense to have, but generally the script decides who broadcasts, and the one who set up the hook should error if it doesn't have the key to do something with it.

Reply:
In the case of the keyed broadcaster the `script.Broadcast`'s `From` field can already be different from the private key in the broadcaster itself. Agree we should error if they don't match - mind if I log this as a ticket?

Reply:
I'll remove the interface method for now to avoid confusion.

Reply:
A ticket for key mismatch with broadcast request is fine. Thanks